vignettes/loo2-mixis.Rmd

```diff
@@ -105,11 +105,11 @@ The _LSVT Voice Rehabilitation Data Set_ (see [link](https://archive.ics.uci.edu
 data(voice)
 y <- voice$y
 X <- voice[2:length(voice)]
-n <- dim(X)[1]
+p <- dim(X)[1]
 p <- dim(X)[2]
 p0 <- 10
 scale_global <- 2*p0/(p-p0)/sqrt(n-1)
-standata <- list(n = n, p = p, X = as.matrix(X), y = c(y), scale_global = scale_global, mixis = 0)
+standata <- list(N = n, P = p, X = as.matrix(X), y = c(y), scale_global = scale_global, mixis = 0)
```

Comment:
This will cause an error because `n` doesn't exist anymore but you have `list(N = n, ...`. It looks like you changed `n` to `p` but now there are two `p`s.

Reply:
Unfortunately the GitHub Actions checks don't catch this error because we don't check all the vignettes because some of them take a long time to run. But this would error if running the vignette locally.

Reply:
fixed